importer/make_KMB_info.py

```diff
@@ -595,8 +598,24 @@ def get_exact_cat_from_name(self, cache):
         """
         exact_match = False
         exact_category_from_name = self.kmb_info.category_exists(self.namn, cache)
-        if exact_category_from_name:
-            exact_category_from_name = pywikibot.Page(self.commons, self.namn)
+
+        if exact_category_from_name is False:
+            return
+
+        # See if we can get an exact cat from church file
```

Comment:
I would move the church_hack out into a separate function.
You could start by checking `if ("Religionsutövning - kyrkor" in self.item_classes)`

Reply:
Done.